Gopkg.lock

```diff
@@ -92,6 +92,14 @@
   revision = "3536a929edddb9a5b34bd6861dc4a9647cb459fe"
   version = "v1.1.2"
 
+[[projects]]
+  branch = "master"
+  digest = "1:a4c2e29d2fb620f39ab706071791242a25df5c62e4793f9b401b07e490d535e9"
+  name = "github.com/otiai10/copy"
```

Comment:
Curious how the lock file changed if the toml file didn't?

Reply:
I just added an import for this lib in source and I think ran `dep ensure`. Looks like maybe I should run `dep ensure -add github.com/otiai10/copy` instead?

Reply:
Well… check out https://github.com/golang/dep/issues/1670. When all else fails, I usually do a fresh `dep init` just to see what's changed 🤷‍♂️